### DISCUSSION TITLE
Whisper CLI compatible client using CTranslate2 and Faster-Whisper

### DISCUSSION THREAD

**Discussion by jordimas** [openai/whisper#1137] [Show and tell]
Hello

I have built a Whisper command line client compatible with original OpenAI client. It uses [CTranslate2](https://github.com/OpenNMT/CTranslate2/) and [Faster-whisper](https://github.com/guillaumekln/faster-whisper) that is up to 4 times faster than openai/whisper for the same accuracy while using less memory.

See: https://pypi.org/project/whisper-ctranslate2/
Github: https://github.com/jordimas/whisper-ctranslate2

I hope that you find it useful

**Comment by mayeaux** [comment]
Thanks I'll definitely be checking this out!

**Comment by kanazei** [comment]
Amazing!
Is it possible to add model large-v1?

**Comment by jordimas** [comment]
Yes, starting from version 0.8 and higher using --model you can select large-v1 and large-v2 as in the OpenAI client.

**Reply by kanazei** [reply]
Thank you! Works when installed via git. Тhe command needs to be corrected:
!pip install git+https://github.com/jordimas/whisper-ctranslate2
But large-v1 and v2 are swapped.

**Reply by jordimas** [reply]
Well, you need "!pip" if you are in a notebook, but this is instead to be run from command line.

Regarding the swap, the models are correct. Thanks

**Reply by NielsMayer** [reply]
The Readme also refers to the wrong pip arguments: the pip commandline is missing the git+http prefix.

**Reply by jordimas** [reply]
Fixed, thanks!

**Comment by mayeaux** [comment]
Works great, I will be changing freesubtitles.ai to use this, thanks a lot!

**Comment by mayeaux** [comment]
Freesubtitles.ai is now running faster-whisper using this CLI, thanks a lot! Works like a charm!
